objects/member.php

```diff
@@ -243,5 +243,62 @@ function send($keep, $private, $text, $fromCountryID=null)
 			$this->userID, $this->gameID, 'Game',
 			$keep, $private, $text, $this->Game->name, $this->gameID);
 	}
+

```

Comment:
Is the movement of this code between files a no-op with regards to any other call sites? Does the old UI still work as it did before regarding civil disorder and rejoining?

Reply:
I tried it and it seems to work as before, but I can't say for sure.

Reply:
I believe it's a no-op because `boardMember` here is a subclass of `Member`